FireRender.Maya.Src/FireRenderIpr.h

```diff
@@ -165,6 +165,8 @@ class FireRenderIpr
 	/** A lock to control access to the system memory frame buffer pixels. */
 	MMutexLock m_pixelsLock;
 
+	MMutexLock m_refreshLock;
```

Comment:
//! \ingroup OpenMaya
//! \brief Methods for managing locks on threads created. 
/*!
 Mutex lock class now just forwarding to std::mutex.
*/
class OPENMAYA_EXPORT OPENMAYA_DEPRECATED(2020, "Use std::mutex directly") MMutexLock

Sounds like we should avoid using MMutexLock

Reply:
Done